public/locales/en/files.json

```diff
@@ -91,7 +91,10 @@
     },
     "step2": {
       "title": "Breadcrumbs",
-      "paragraph1": "The current work directory, you can navigate through the folder hierarchy by clicking on them."
+      "paragraph1": "These breadcrumbs display the path to your current directory. You can…",
+      "bullet1": "Click an individual breadcrumb to go to that directory",
+      "bullet2": "Right-click a breadcrumb to get the actions menu for that directory",
```

Comment:
```suggestion
```

We'll need to remove this Joyride hint if we don't have context menu yet.

Reply:
We have the context menu, what's not possible is to right click on another breadcrumb WHILE the context menu is open :)

Reply:
@rafaelramalho19 apologies, I misunderstood the comment chain. Never mind 😊